src/app/swap/layout.tsx

```diff
@@ -1,11 +1,11 @@
-import './globals.css'
```

Comment:
This change looks a lil weird to me: src/app/layout.tsx → src/app/swap/layout.tsx

Cause we still need the html, header, footer etc. (replace footer with new one)

Probably only the inner <Flex> part should move into the swap layout file.

Reply:
Moved the shared items into a common root layout, but let's add the header/footer as part of a future PR. We need the header to be wrapped by `Providers` since it uses RainbowKit, but using the ChakraProvider caused a few unexpected styling changes, which is why I ended up creating `ProvidersLite`.